src/ralph/data_center/models/networks.py

```diff
@@ -529,15 +514,7 @@ def save(self, allow_device_change=True, *args, **kwargs):
         if settings.CHECK_IP_HOSTNAME_ON_SAVE:
             if not self.address:
                 self.address = network.hostname(self.hostname, reverse=True)
-            if not self.hostname:
-                self.hostname = network.hostname(self.address)
         self.number = int(ipaddress.ip_address(self.address))
-        try:
-            self.network = Network.from_ip(self.address)
```

Comment:
remove this method from Network model too - if we'll need this in the future we'll copy it from Ralph 2.0 - for now it's useless and unused code.